sys/net/application_layer/dns/dns.c

```diff
@@ -28,6 +28,9 @@
 /* min domain name length is 1, so minimum record length is 7 */
 #define DNS_MIN_REPLY_LEN   (unsigned)(sizeof(sock_dns_hdr_t ) + 7)
 
+/* global DNS server UDP endpoint */
+sock_udp_ep_t sock_dns_server;
```

Comment:
is one global DNS server sufficient?

Reply:
I hoped that this could be const for some use-cases, but I'm OK with moving it here for now.

Reply:
> is one global DNS server sufficient?

For now it should be. Your local LAN usually also is happy with just one ;-).